library/core/src/ptr/mod.rs

```diff
@@ -753,9 +754,13 @@ pub const unsafe fn read<T>(src: *const T) -> T {
 /// As a result, using `&packed.unaligned as *const FieldType` causes immediate
 /// *undefined behavior* in your program.
 ///
+/// Instead you must use the [`ptr::addr_of!`](crate::ptr::addr_of) macro to
```

Comment:
This is in the `ptr` module, you don't need `crate::ptr` again.
```suggestion
/// Instead you must use the [`ptr::addr_of!`](addr_of) macro to
```